tree/dataframe/src/RDataFrame.cxx

```diff
@@ -456,6 +456,40 @@ executed at the moment they are called, but they are **lazy**, i.e. delayed unti
 accessed through the smart pointer. At that time, the event loop is triggered and *all* results are produced
 simultaneously.
 
+For yet another example of the difference between the correct and incorrect running of the event-loop, see the following
```

Comment:
I would add a paragraph title here, maybe like `How to properly exploit RDataFrame lazyness` or something like that